utils/logalloc.cc

```diff
@@ -2894,10 +2894,10 @@ void allocating_section::on_alloc_failure(logalloc::region& r) {
     r.allocator().invalidate_references();
     if (r.get_tracker().get_impl().segment_pool().allocation_failure_flag()) {
         _lsa_reserve *= 2;
-        llogger.debug("LSA allocation failure, increasing reserve in section {} to {} segments", fmt::ptr(this), _lsa_reserve);
+        llogger.info("LSA allocation failure, increasing reserve in section {} to {} segments; trace: {}", fmt::ptr(this), _lsa_reserve, current_backtrace());
```

Comment:
I hope we don't get here too frequently since `current_backtrace()` is pretty expensive.

Reply:
It's hard to predict. It's exponential so growing from 1 segment to 8 segments (1MB) is just 3 traces.

The std reserve theoretically shouldn't grow at all, since we have the background reclaimer working to maintain 60MB of reserves.

Reply:
Each memtable has its own allocating sections, which reset after memtable flush. So this can continuously trigger. With tablets this will be worse because we will have more and smaller memtables.

Reply:
Yes, I thought of having new memtables inherit the configuration of old memtables. Maybe it should be a prerequisite.

Reply:
It makes sense to inherit.